service-registry/src/main/java/io/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -122,6 +127,9 @@ private boolean checkSchemaIdSet() {
           microservice.getVersion(),
           localSchemas,
           existSchemas);
+      if (!serviceCenterEnvIsDev()) {
```

Comment:
mode determined by SC?
i remember changed design that determined by microservice?